admin/class-ccb-core-cpts.php

```diff
@@ -310,6 +310,10 @@ public static function get_calendar_custom_fields_map() {
 				'api_mapping' => 'event_duration',
 				'data_type' => 'int',
 			),
+			'calendar_location' => array(
```

Comment:
@nicholaspetersen thanks for the PR, this looks great. i'm actually in the process of cleaning up some of this code and fixing some bugs so i'll merge this in with my changes.

in the mean time, if you want, you can use a filter to adjust these definitions (see https://github.com/jaredcobb/ccb-core/blob/1a6e88b47ad7d5293e88bac277e72cbc4e33a602/admin/class-ccb-core-sync.php#L669)

for example, in your theme / plugin you can do something like

```
function my_theme_add_ccb_location( $calendar_custom_fields_map ) {
	$calendar_custom_fields_map['calendar_location'] = array(
		'api_mapping' => 'location',
		'data_type' => 'string',
	);
	return $calendar_custom_fields_map;
}
add_filter( 'ccb_get_calendar_custom_fields_map', 'my_theme_add_ccb_location', 10, 1 );
```